docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.9"
+
+services:
+
+  as:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+      - "8000:8000"
+    extra_hosts:
+      - "hostnet:192.168.112.1"
```

Comment:
potremmo usare un environment variable?

Reply:
Sì, possiamo metterle. Il file che docker-compose legge di default è il ".env" che però attualmente è in .gitignore: o lo rimuoviamo dal .gitignore altrimenti chiamiamo il file di enviroment in modo differente e poi usiamo docker-compose con l'opzione apposita, per esempio `docker-compose --env-file ./docker-compose.env up`. La prima opzione mi sembra più semplice.

Quali e quante variabili vuoi creare esattamente?

Reply:
tutte quelle che possono tornare utili in produzione.
sta bene compose env, l'abbiamo nel README?

basterebbe questo e un esempio di questo

Reply:
Ho aggiunto il file `docker-compose.env` e aggiornato il README.md per utilizzarlo. Risolvo la conversazione ora "Outdated", se c'è altro segnalamelo pure nel codice aggiornato.